.eslintrc

```diff
@@ -0,0 +1,27 @@
+{
+    "env": {
+        "browser": true,
+        "amd": true,
+        "node": true
+    },
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
+    "extends": [
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:prettier/recommended",
+        "prettier/@typescript-eslint"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 2020,
+        "sourceType": "module"
+    },
+    "ecmaFeatures": { "jsx": true },
```

Comment:
I think ecmaFeatures has to be inside parserOptions.
https://eslint.org/docs/user-guide/configuring

Reply:
Thanks, fixed in 6e66da9d377e7dbd1846ab86eb016da26ef1b156